SPECS/azure-iot-sdk-c/azure-iot-sdk-c.spec

```diff
@@ -23,7 +23,7 @@ Distribution:   Azure Linux
 #           --owner=0 --group=0 --numeric-owner \
```

Comment:
Could you still update the `LTS_08_2023` in the `1. git clone --recursive --single-branch --branch LTS_08_2023` line above? I can't mark the correct line in GitHub, unfortunately.